templates/openstack/compare.html

```diff
@@ -400,6 +421,8 @@ <h2>Contact us</h2>
   </div>
 </section>
 
+{% with first_item="_private_cloud_contact_us", second_item="_cloud_newsletter", third_item="_cloud_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
```

Comment:
@bethcollins92 why this contextual footer, not the more OpenStack based ones? like:

```
  {% with first_item="_openstack_next_step", second_item="_cloud_newsletter", third_item="_openstack_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
```

either way, you need to edit the `_openstack_further_reading.html` file to have 2 tagIds like:

```
{% with limit="5", tagId="1327,1350" %}
  {% include "shared/contextual_footers/_further_reading.html" %}
{% endwith %}
```

Reply:
Thanks @pmahnke, updated